### ISSUE TITLE
Same-origin policy using hot loader inside an iframe.

### ISSUE THREAD

**Issue by gdborton** [gaearon/react-hot-loader#114]
I'm getting the below error when running webpack-dev-server and react-hot-loader inside of an iframe of a different protocol/port/domain.

My use case is parent: localhost:4100 and child: localhost: 3000

```
Uncaught SecurityError: Failed to read the 'frame' property from 'Window':
Blocked a frame with origin "https://localhost:3000" from accessing a frame with
origin "https://localhost:4100". Protocols, domains, and ports must match.
```

Updating [this line](https://github.com/gaearon/react-hot-loader/blob/master/makeExportsHot.js#L32) to the below, makes it work for me.
`if (key === 'frameElement' || !isReactClassish(freshExports[key], React)) {`

**Comment by gaearon** [comment]
Can `try/catch` help here? We wouldn't want to special-case any specific properties that might throw in some rare cases.

**Comment by gdborton** [comment]
That should work as well.  Did you want to do anything specific with the exception?

**Comment by gaearon** [comment]
I think we may silently ignore it, or maybe warn unless it's too noisy.

**Comment by gaearon** [comment]
Should be fixed in 1.2.9.
